.github/ci.md

```diff
@@ -68,7 +70,7 @@ If they don't match, it describes how to make the updates to include the changes
 - integration
 
 ### Cancelling Workflow
-[This](./workflows/cancel-ci.yaml) workflow is triggered each time a [build-branch](ci.md#build-branch-workflow) workflow is triggered.  It reduces GA usage, by cancelling PR workflows that have a job failure.  Specifically, it checks all PR workflows running at the time it is invoked. It cancels any of those which have a failed job, (at that time). Any PR workflows with jobs that fail after that will be caught by a subsequent run of the "Cancelling" workflow.
+[This](./workflows/cancel-ci.yaml) workflow is triggered each time a [full-ci](ci.md#full-ci-workflow) workflow is triggered.  It reduces GA usage, by cancelling PR workflows that have a job failure.  Specifically, it checks all PR workflows running at the time it is invoked. It cancels any of those which have a failed job, (at that time). Any PR workflows with jobs that fail after that will be caught by a subsequent run of the "Cancelling" workflow.
```

Comment:
Don't see this workflow in the ./workflows directory : `./workflows/cancel-ci.yaml`.